src/goto-unwind/src/goto_unwind_parse_options.h

```diff
@@ -0,0 +1,23 @@
+/// \file
+/// \author Diffblue Ltd.
+
+#ifndef CPROVER_GOTO_UNWIND_SRC_GOTO_UNWIND_PARSE_OPTIONS_H
+#define CPROVER_GOTO_UNWIND_SRC_GOTO_UNWIND_PARSE_OPTIONS_H
+
+#include <string>
+
+#include <goto-programs/goto_model.h>
+#include <util/parse_options.h>
+
+class goto_unwind_parse_optionst : public parse_options_baset
+{
+public:
+  goto_unwind_parse_optionst(int argc, const char **argv);
+
+  int doit() override;
+  void help() override;
+  goto_modelt get_goto_model(const std::string &goto_binary_path);
+  void do_unwind(goto_modelt &model, size_t limit);
```

Comment:
:pick: The `get_goto_model` and `do_unwind` functions have been made part of the public interface, but they are only used from inside the `goto_unwind_parse_optionst` class and they are missing doxygen. I think these should be made `protected` at the least, but I would probably have written them as free functions myself, in order to avoid implicitly passing them the entire state of the class.

Reply:
Well, the main advantage of having them be member functions is to have `log` immediately available and have a namespace for them. But I’ll make them private.